docs/docs/concepts/jobs.md

```diff
@@ -66,16 +46,60 @@ class JobHandler(JobHandlerInterface):
 
 ```
 
+
+
+
 A few important things to note:
-- A defined variable called `_SUPPORTED_TYPE` decides what JobHandler to use for a given job. This is the same as the runner type in the job entity.
+- A defined variable called `_SUPPORTED_TYPE` decides what JobHandler to use for a given job entity. 
+This is the same as the runner type in the job entity.
 - The JobHandlerInterface and Job class are defined in dm-job repository.
 - To access the job entity, use `self.job.entity`
 - - `self.job.entity.applicationInput` is used for supplying inputs to the job. For example, you can include inputs to calculations.
 - - `self.job.entity.outputTarget` is used for defining where the result of the job should be stored.
+- The file must have name `__init__.py`
+
+
+Also, the file must be available as a file in the dm-job container.
+For example, if you want to add a new job to the [dm-core-packages repository](https://github.com/equinor/dm-core-packages), you would create your JobHandler file inside the dm-core-packages repository and make it available for the dm-job API.¨¨
+
+This can be achieved by using docker volumes. 
+For example, in the [docker-compose file in dm-core-packages](https://github.com/equinor/dm-core-packages/blob/main/example/docker-compose.yaml), 
+docker volume is used to "copy" the content of `dm-core-packages/example/job_handlers` into the folder `dm-job/src/job_handler_plugins`.
```

Comment:
This could be rephrased. Perhaps start by saying that this is done automatically if you place the init file in the correct folder, and then explain the logic of how that works afterwards